authentication improved (#16)
playcanvas/vscode-extension: README.md

## Usage

* Download and install extension in VS Code.
* Generate an access token on your [PlayCanvas account page](https://playcanvas.com/account) - check [this document](https://developer.playcanvas.com/en/user-manual/api/#authorization) for details, copy it to clipboard

> * Select "PlayCanvas: Add Project" command in VS Code command menu, paste your token into a message box

Reviewer: ```suggestion
* Select "PlayCanvas: Add Project" command in VS Code command menu and paste your token into the message box
```

> * Work with your project in VS Code workspace

Reviewer: ```suggestion
* Work with your project in VS Code workspace.
```

#### Settings

> The extension has just 1 setting - usePlaycanvasTypes to add types support. Access Token is requested when you are adding a project.

Reviewer: ```suggestion
The extension has just 1 setting: `usePlaycanvasTypes` (to add types support). An Access Token is requested when you are adding a project.
```

## Requirements

* An existing PlayCanvas account with an access token generated.

## Extension Settings

* `playcanvas.usePlaycanvasTypes`: Automatically adds a reference to PlayCanvas types files for code suggestions. Line is not saved. Default is true.

> Playcanvas Access Token is requested when you add a project - generate an access token on your [account page](https://playcanvas.com/account).

Reviewer: ```suggestion
A PlayCanvas Access Token is requested when you add a project. Generate an access token on your [account page](https://playcanvas.com/account).
```
